src/test/kotlin/no/nav/syfo/testutil/HttpClientTest.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.syfo.testutil
+
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.mock.MockEngine
+import io.ktor.client.engine.mock.respond
+import io.ktor.client.features.json.JacksonSerializer
+import io.ktor.client.features.json.JsonFeature
+import io.ktor.http.Headers
+import io.ktor.http.HttpStatusCode
+import io.ktor.http.headersOf
+
+data class ResponseData(val httpStatusCode: HttpStatusCode, val content: String, val headers: Headers = headersOf("Content-Type", listOf("application/json")))
+
+fun getSerializer(): JacksonSerializer {
+    return JacksonSerializer {
+        registerKotlinModule()
+    }
+}
+
+class HttpClientTest {
```

Comment:
Hva tester vi her egentlig..? :)

Reply:
Det er egentlig bare en helper klasse for å gjøre testing lettere der jeg brukte httpClient